bin/check-a11y.js

```diff
@@ -23,102 +23,181 @@ const pkgFilter = hasFilter
+ * https://github.com/dequelabs/axe-core/blob/master/doc/API.md
+ */
 const checkA11y = async () => {
   const driver = new Builder()
     .forBrowser('chrome')
     .setChromeOptions(new Options().addArguments('-headless'))
     .build()
 
-  let testedStory
+  let testedComponent
 
-  for (const storyId of storiesList) {
-    if (testedStory !== stories[storyId].title)
-      console.log(`Checking accessibility for ${stories[storyId].title}...`)
+  for (const docId of docsList) {
+    if (testedComponent !== entries[docId].title) {
+      console.log(`Checking accessibility for ${entries[docId].title}...`)
+    }
 
-    testedStory = stories[storyId].title
+    testedComponent = entries[docId].title.toLowerCase()
 
     try {
-      await driver.get(`http://localhost:6006/iframe.html?viewMode=story&id=${storyId}`)
+      await driver.get(`http://localhost:6006/?path=/docs/${docId}`)
 
       const results = await new AxeBuilder(driver)
         .options({
           resultTypes: ['violations', 'incomplete'],
           rules: {
+            // 'meta-viewport': { enabled: false },
             'page-has-heading-one': { enabled: false },
             'landmark-one-main': { enabled: false },
             region: { enabled: false },
             'color-contrast': { enabled: false },
           },
           runOnly: ['best-practice', 'wcag2a', 'wcag2aa', 'wcag21a', 'wcag21aa'],
-          reporter: 'no-passes',
+          reporter: 'v2',
         })
-        .exclude('.docblock-argstable')
+        .exclude('*:not(.docs-story)')
         .analyze()
 
-      if (results.violations.length) {
-        issues = results.violations.reduce(
-          (acc, cur) => {
-            acc[cur.impact] += cur.nodes.length
-            return acc
-          },
-          {
-            minor: 0,
-            moderate: 0,
-            serious: 0,
-            critical: 0,
-          }
-        )
-
-        const totalIssues = Object.values(issues).reduce((a, b) => a + b, 0)
-        console.log(chalk.red(`Found ${totalIssues} ${stories[storyId].title} issue(s)):`))
-
-        results.violations.forEach(issue => {
-          console.log(
-            impactColor[issue.impact](`- [${issue.impact}] ${issue.help} (${issue.nodes.length})`)
-          )
-        })
+      const duplicate = report[testedComponent]
+      const { [testedComponent]: _, ...rest } = report
+
+      const { timestamp, url, incomplete, violations } = results
+
+      report = {
+        ...rest,
+        [testedComponent]: {
+          timestamp,
+          ...(duplicate ? { url: [...duplicate.url, url] } : { url: [url] }),
+          ...(duplicate
+            ? { incomplete: mergeResultArrays(incomplete, duplicate.incomplete) }
+            : { incomplete: mergeResultArrays(incomplete) }),
+          ...(duplicate
+            ? { violations: mergeResultArrays(violations, duplicate.violations) }
+            : { violations: mergeResultArrays(violations) }),
```

Comment:
```suggestion
          timestamp,
          url: duplicate ? [...duplicate.url, url] : [url],
          incomplete: duplicate ? mergeResultArrays(incomplete, duplicate.incomplete) : mergeResultArrays(incomplete),
          violations: duplicate ? mergeResultArrays(violations, duplicate.violations) : mergeResultArrays(violations),
```

is not the same?

Reply:
Yes it is the same, but I find this less readable.